NALU Compile Error with GCC 7.2.0

I am trying to compile NALU-CFD on POWER8 systems with GCC 7.2.0. The version is from today. Just wanted to let you know what has come up.

```
/ascldap/users/sdhammo/git/nalucfd-github-repo/src/PecletFunction.C:113:16: error: duplicate explicit instantiation of ‘class sierra::nalu::ClassicPecletFunction<double>’ [-fpermissive]
 template class ClassicPecletFunction<DoubleType>;
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/ascldap/users/sdhammo/git/nalucfd-github-repo/src/PecletFunction.C:114:16: error: duplicate explicit instantiation of ‘class sierra::nalu::TanhFunction<double>’ [-fpermissive]
 template class TanhFunction<DoubleType>;
                ^~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/nalu.dir/src/PecletFunction.C.o] Error 1
make[2]: Leaving directory `/home/sdhammo/git/nalucfd-github-repo/build-gcc-720'
make[1]: *** [CMakeFiles/nalu.dir/all] Error 2
make[1]: Leaving directory `/home/sdhammo/git/nalucfd-github-repo/build-gcc-720'
make: *** [all] Error 2
```

--- a/src/PecletFunction.C
+++ b/src/PecletFunction.C
@@ -110,8 +110,11 @@ T TanhFunction<T>::execute(const T indVar)
 
 template class ClassicPecletFunction<double>;
 template class TanhFunction<double>;
+
+#ifdef STK_HAVE_SIMD
 template class ClassicPecletFunction<DoubleType>;
 template class TanhFunction<DoubleType>;
+#endif
 
 } // namespace nalu
 } // namespace Sierra

